CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # nf-core/mhcquant: Changelog
 
+## v1.3 nf-core/mhcquant "Red Parrot" - 2019/07/19
+
+### `Fixed`
```

Comment:
Provide bullets for these changes in the entire `CHANGELOG.md` otherwise they are all listed on the same line.

Reply:
done